src/custom/simplifions/components/SimplifionsDataApiCard.vue

```diff
@@ -55,11 +55,10 @@
 
 <script setup lang="ts">
 import DataserviceCard from '@/components/DataserviceCard.vue'
-import type { DataserviceV2 } from '@/model/dataservice'
 import DatagouvfrAPI from '@/services/api/DatagouvfrAPI'
 import { fromMarkdown } from '@/utils'
-import type { DatasetV2 } from '@datagouv/components'
-import { DatasetCard } from '@datagouv/components'
+import type { Dataservice, DatasetV2 } from '@datagouv/components-next'
```

Comment:
I have one small complaint with the new DatasetV2 component : It's not giving me a `target="_blank_"` link, which I need. I have a list of cards with both Datasets and Dataservices in it, and I want them all to open new tabs.

With this change, I'm getting `target="_blank"` links for Dataservices, but not for Datasets.

<img width="1365" height="585" alt="image" src="https://github.com/user-attachments/assets/20dc1aa8-d7a5-480d-908d-c20107caea4b" />

:point_up: This example is on the page `/cas-d-usages/formation-professionnelle-financee` of the `simplifions` vertical.

Reply:
(The tiny "open in a new tab" symbol top right of a card)

Reply:
I'll add a new `props` to do it 👍🏻

Before it was based on the link being external or not but we removed this coupling.

Reply:
Unrelated but : I was using a custom component for DataserviceCard mainly because I wanted my list of both dataservices and datasets to be harmonious looking. That means having the picture of the orga on the left on the DataserviceCard.

Would you consider adding this picture to the generic DataserviceCard, and making it accessible in the lib ? then we could completely remove the custom component in udata-frontkit, I think.

Reply:
The `DataserviceCard ` is available in the new package, but I agree with you that the current design isn't suitable for this usecase.
That's why I didn't replace the custom dataservice component. I'm waiting for a reply from our PO on your issue to know if we want to implement it or not.

Reply:
What is done exactly ? 

I still see the datasets cards with "non external" links, on this branch, in the page http://localhost:5173/cas-d-usages/formation-professionnelle-financee

If I should make a change to use the new props or something, feel free to ask me to do it